pkgs/by-name/ra/rancid/package.nix

```diff
@@ -0,0 +1,110 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, writeShellScriptBin
+, autoreconfHook
+, libtool
+, makeBinaryWrapper
+, coreutils
+, expect
+, git
+, gnugrep
+, inetutils # for telnet
+, gnused
+, openssh
+, perl
+, runtimeShell
+}:
+
+# we cannot use resholvePackage yet - the scripts are too hairy
+
+let
+  inherit (lib) concatStringsSep makeBinPath mapAttrsToList replaceStrings;
+
+  # The installer executes ping to figure out how to call it and then sets the
```

Comment:
If this `ping` script is meant for the installer, how come we can't be sure whether `/run/wrappers/bin/ping` exists or not? That test in the script is a bit confusing... Reading the script it feels as if it is meant to be a `ping` wrapper that the binaries produced by the derivation need to use as a `ping` executable, but this first line seems to tell otherwise.